content/FAQ.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Frequently Asked Questions
+url: FAQ/
+summary: Frequently asked questions
+showToc: true
+tocopen: true
+---
+
+## I read the docs but still need help!
+
+Come join the [discord](https://discord.gg/PeHR6Tjyvn)! Let the community help you in real-time!
+
+## What is VPupPr
+
+VPupPr is an open-source project that animates 3D objects using the glb or vrm file extension. 
+Most notable for using 3D vtube models as its main usage.
```

Comment:
Sentence feels awkward, could be reworded as: It is most notable for its use as a tracker and renderer for VTuber models.

"Most notably" could be used, but the adjective is supposed to allude to something mentioned in the previous sentence. In this case, there is nothing that it is contrasting with or expanding on that relates to notability.